homework/dice.js

```diff
@@ -0,0 +1,11 @@
+function rollDice()
+{
+  var rand = Math.round(Math.random() * 6);
+   var rand2 = Math.round(Math.random() * 6);
```

Comment:
There is a chance of getting `0` as a random number here, we only want random numbers from 1 to 6, how would you make sure that your program doesn't output `0`?